comment/src/README.md

```diff
@@ -0,0 +1,10 @@
+This extension provides commands to manipulate comments in source code.
+
+
+
+``` javascript
```

Comment:
What is this for?

The docs won't build unless this file either doesn't exist or has markers like `@toggleLineCommentCmd` for every export in the package. Probably easiest to remove it for now.

Reply:
Perfect. I'll remove it for now then.